src/midi/midi_mt32.cpp

```diff
@@ -0,0 +1,516 @@
+	        "Nicer overdrive characteristics than the DAC hacks (it simply clips samples within range)\n"
+	        "Much less likely to overdrive than any other mode.\n"
+	        "Half the volume of any of the other modes.\n"
+	        "Perfect for developers while debugging :)\n\n"
+
+	        "GENERATION1 = 2\n"
+	        "Re-orders the LA32 output bits as in early generation MT-32s (according to Wikipedia).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number, and XX means the bit is always low):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 XX\n\n"
+
+	        "GENERATION2 = 3\n"
+	        "Re-orders the LA32 output bits as in later generations (personally confirmed on my CM-32L - KG).\n"
+	        "Bit order at DAC (where each number represents the original LA32 output bit number):\n"
+	        "15 13 12 11 10 09 08 07 06 05 04 03 02 01 00 14");
+
+	const char *mt32analogModes[] = {"0", "1", "2", "3", 0};
+	int_prop = sec_prop.Add_int("analog", when_idle, 2);
+	int_prop->Set_values(mt32analogModes);
+	int_prop->Set_help(
+	        "MT-32 analogue output emulation mode\n"
+	        "Digital = 0\n"
+	        "Only digital path is emulated. The output samples correspond to the digital output signal appeared at the DAC entrance.\n"
+	        "Fastest mode.\n\n"
+
+	        "Coarse = 1\n"
+	        "Coarse emulation of LPF circuit. High frequencies are boosted, sample rate remains unchanged.\n"
+	        "A bit better sounding but also a bit slower.\n\n"
+
+	        "Accurate = 2 - default\n"
+	        "Finer emulation of LPF circuit. Output signal is upsampled to 48 kHz to allow emulation of audible mirror spectra above 16 kHz,\n"
+	        "which is passed through the LPF circuit without significant attenuation.\n"
+	        "Sounding is closer to the analog output from real hardware but also slower than the modes 0 and 1.\n\n"
+
+	        "Oversampled = 3\n"
+	        "Same as the default mode 2 but the output signal is 2x oversampled, i.e. the output sample rate is 96 kHz.\n"
+	        "Even slower than all the other modes but better retains highest frequencies while further resampled in DOSBox mixer.");
+
+	const char *mt32reverbModes[] = {"0", "1", "2", "3", "auto", 0};
+	str_prop = sec_prop.Add_string("reverb.mode", when_idle, "auto");
+	str_prop->Set_values(mt32reverbModes);
+	str_prop->Set_help("MT-32 reverb mode");
+
+	const char *mt32reverbTimes[] = {"0", "1", "2", "3", "4",
+	                                 "5", "6", "7", 0};
+	int_prop = sec_prop.Add_int("reverb.time", when_idle, 5);
+	int_prop->Set_values(mt32reverbTimes);
+	int_prop->Set_help("MT-32 reverb decaying time");
+
+	const char *mt32reverbLevels[] = {"0", "1", "2", "3", "4",
+	                                  "5", "6", "7", 0};
+	int_prop = sec_prop.Add_int("reverb.level", when_idle, 3);
+	int_prop->Set_values(mt32reverbLevels);
+	int_prop->Set_help("MT-32 reverb level");
```

Comment:
We hid reverb setting in FluidSynth and there were 0 complaints - I guess we should hide the reverb setting in here as well.

Reply:
I agree.